fns-hl7-pipeline/fn-receiver-debatcher/src/main/kotlin/gov/cdc/dex/hl7/Function.kt

```diff
@@ -262,10 +267,10 @@ class Function {
     private fun getValueOrNullString(
         metaDataMap: Map<String, String?>,
         keysToTry: List<String>
-    ): String? {
+    ): String {
         val value = getValueOrDefaultString(metaDataMap, keysToTry)
         return if (value == "UNKNOWN") {
-            null
+            ""
```

Comment:
What's the reason for returning an empty string instead of null?

Reply:
updated to ""  because of mvn errors. Now, using default method instead of null.